src/common/dataAdaptors/siteAdaptors.js

```diff
@@ -13,7 +13,6 @@ import {
   getMediaPath,
   selectOneMediaDataHelper,
 } from 'common/utils/mediaHelpers'
-import placeholder from 'images/cover-thumbnail.png'
```

Comment:
NB: Removing the backup logo from this adaptor as it is not successfully being loaded on main. Will address how we handle back-up logos in this future ticket
https://firstvoices.atlassian.net/browse/FW-4846